app/javascript/react/components/Reviews/ReviewForm.js

```diff
@@ -0,0 +1,103 @@
+import React, {useState } from 'react';
+import ErrorList from '../Artists/ErrorList';
+
+const ReviewForm = ({ addReview }) => {
+    const [errors, setErrors] = useState({});
+    const [addNewReview, setAddNewReview] = useState({
+        rating:"",
+        title: "",
+        body: ""
+    })
+
+    const handleInputChange = (e) => {
+        setAddNewReview({
+          ...addNewReview,
+          [e.currentTarget.id]: e.currentTarget.value,
+        });
+      };
+
+    const validForSubmission = () => {
+        let submitErrors = {};
+        const requiredFields = ["rating","title", "body"];
+        requiredFields.forEach((field) => {
+          if (addNewReview[field].trim() === "") {
+            submitErrors = {
+              ...submitErrors,
+              [field]: "is blank",
+            };
+          }
+        });
+    
+        setErrors(submitErrors);
+        return _.isEmpty(submitErrors);
+    };
+      
+    const clearForm = (e)=> {
+        e.preventDefault();
+        setAddNewReview({
+            rating: "",
+            title: "",
+            body: ""
+          });
+          setErrors({});
+      }
+        
+      const onSubmitHandler = (e) => {
+          e.preventDefault();
+          if (validForSubmission()) {
+              addReview({review: addNewReview});
+              setAddNewReview({
+                  rating: "",
+                  title: "",
+                  body: ""
+                });
+            }
+      };
```

Comment:
correct indentation for this method
remove the semicolon on line 46

Your app should either use semicolons all the time, or not at all. In this case: not at all

Reply:
line 53 as well